src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -49,7 +51,10 @@ That will eventually go away.
 
 `chrome` should point to your installation of Chrome. The exact location will
 vary from platform to platform. Since I'm on Mac, I created convenient aliases
-for each version of Chrome that I have installed:
+for each version of Chrome that I have installed.
+
+If you're on the stable channel of Chrome and cannot get the Beta, I recommend
+using `chrome-canary` until Chrome 59 hits stable:
```

Comment:
Link to Canary link. Also peeps might not know what Stable, Canary, etc. are.

"Go to `chrome://help` to see what version of Chrome you're running. If it's not version 59, download [Canary](canary), the developer preview version of Chrome."